patches/chromium/revert_remove_contentrendererclient_shouldfork.patch

```diff
@@ -0,0 +1,188 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Wed, 23 Oct 2019 15:30:18 -0700
+Subject: Revert "Remove ContentRendererClient::ShouldFork."
+
+This reverts commit 6b068eb8ca4a3c7350bdafa22fc0cf0636ef8b74.
```

Comment:
Filed https://github.com/electron/electron/issues/20735 to track this

Reply:
What breaks if we stop handling `ShouldFork`?

Reply:
We want the browser process to manage all navigations in the non-sandbox mode so that the process fork works as expected, this was handling that. We should start using NavigationThrottlers as an alternative, I am working on it now.